icons/mojangstudios.svg

```diff
@@ -1 +1 @@
8.17 428.17 0 01-4.6454 0zm10.3814 8.997a1.3092 1.3092 0 101.321 1.3165 1.329 1.329 0 00-1.3004-1.3164 1.3092 1.3092 0 00-.0206 0zm3.8498 0c-.5953-.008-.9903.3577-.9559.855.0227.3182.2274.493.4893.6173.1616.0732.3379.1258.4944.21a.1828.1828 0 01.0944.1944.2004.2004 0 01-.1661.1353.512.512 0 01-.3372-.1418c-.1404-.136-.2508-.131-.3838-.01a.8974.8974 0 01-.1464.104c-.1463.0848-.155.1732-.0365.299.3584.3833 1.0275.471 1.467.1836.4901-.321.4886-.9779-.0124-1.277a4.6281 4.6281 0 00-.5697-.2633c-.1067-.0454-.2136-.1097-.1375-.2245.0373-.0556.1273-.1046.2691-.0556a.3284.3284 0 01.1266.087c.1258.1397.2487.142.3832.0124a1.3077 1.3077 0 01.1792-.1353c.1317-.087.1105-.1725.0095-.2713a1.1702 1.1702 0 00-.7672-.3196zm-22.123.003a.847.847 0 00-.2427.0363 1.4693 1.4693 0 00-.3577.1463.6904.6904 0 00-.0285 1.1585 2.2373 2.2373 0 00.3964.2062c.0943.0446.207.0606.2882.1206a.2194.2194 0 01.106.2078.199.199 0 01-.1909.1316.5946.5946 0 01-.3138-.1463c-.139-.1345-.2501-.1294-.3927.0102a.5485.5485 0 01-.1288.0914c-.1535.0805-.16.1624-.0657.2655a1.1432 1.1432 0 00.9917.379c.4966-.0468.8045-.3233.8637-.741.0527-.3554-.139-.6136-.6173-.8272-.1228-.0548-.256-.1016-.37-.1601a.1463.1463 0 01-.0732-.106.1148.1148 0 01.0425-.1288.2786.2786 0 01.122-.041c.082-.01.1684.0659.229.1259.1178.1155.2195.1177.3357.016a2.1042 2.1042 0 01.2092-.1595c.1163-.0789.1185-.1564.0205-.2559-.1843-.188-.4067-.2802-.708-.3219a.847.847 0 00-.1156-.008zm14.7491.0692c-.2143-.007-.2925.0607-.2874.2786.011.5544 0 1.1095.0043 1.664 0 .6194-.0577.5155.5479.5302.1645.004.2289-.0541.2245-.2194-.0088-.3387-.003-.6766-.003-1.0138v-.7423c-.0051-.5559.0731-.4805-.4863-.4973zm-8.1385.001c-.136 0-.1946.0578-.1924.1864.0095.512 0 1.024.0359 1.5302a.833.833 0 00.7218.7832 2.35 2.35 0 00.6246.007c.4213-.052.7073-.3124.7797-.7278a2.6169 2.6169 0 00.0534-.6457 2.276 2.276 0 000-.2502c-.0022-.1484-.0015-.2984-.0022-.449 0-.4746.0498-.4307-.4388-.4315-.302 0-.302 0-.3035.2977 0 .3906.0066.7811-.0066 1.1702-.0088.2538-.1418.3795-.3561.3795-.2143 0-.3314-.1103-.3446-.3583-.0168-.3225-.0123-.6459-.0175-.9684-.0095-.6122.0731-.512-.5537-.5237zm-3.764.0008c-.2962 0-.2962 0-.2976.3056 0 .3555 0 .3554.3599.3562h.1572c.155.007.2143.0476.2113.2048-.008.3657 0 .7365 0 1.1044 0 .5748-.0446.485.512.501.1902.006.2523-.0688.2494-.2487-.0059-.4433 0-.8864 0-1.3296 0-.1704.0388-.2334.2319-.2318h.158c.343 0 .3437 0 .3415-.3533a.936.936 0 01-.0044-.1112c.0037-.1463-.0621-.2004-.2062-.1974-.3006.007-.6012 0-.901 0zm7.8202 0c-.174-.009-.2442.0548-.239.2324.0087.3306 0 .6583 0 .9918v1.0371c0 .136.0423.2122.1937.2085.308-.007.6173.011.9238-.0124.6582-.0497 1.1204-.4945 1.1585-1.081.0497-.7401-.2816-1.2068-1.0042-1.3355-.3357-.06-.6883-.0241-1.0328-.041zm.6104.6713c.5286-.0214.786.2899.6455.7657-.0732.256-.409.4447-.662.3657a.1463.1463 0 01-.0884-.103c-.0066-.3358-.0045-.6722-.0045-1.0196.0376-.004.074-.007.1093-.009zm7.0038.0121a.5464.5464 0 01.5497.5445.5259.5259 0 01-.5515.5515.5288.5288 0 01-.5413-.5397.5464.5464 0 01.543-.5563z"/></svg>
```

Comment:
<details>
<summary>There seem to be some differences between this (red) and the SVG source found (black).</summary>
<img src="https://user-images.githubusercontent.com/1830867/102913837-3ff8d080-4477-11eb-843a-bf2120c11921.png"/>
</details>
<details>
<summary>Even with the PNG source (below), there are some small differences.</summary>
<img src="https://user-images.githubusercontent.com/1830867/102914531-594e4c80-4478-11eb-83a6-1b25088541e3.png"/>
</details>
<details>
<summary>I've just pulled in the Gamepedia SVG, and it looks like the best possible match to the PNG file linked as source.</summary>
<img src="https://user-images.githubusercontent.com/1830867/102915002-fdd08e80-4478-11eb-8831-2134b7ce9c5e.png"/>
</details>

If there's no objection from any other @simple-icons/maintainers - I'd recommend that we use the **Gamepedia SVG** as our source - and can review at a later date, should the logo change and they can't keep up with changes.

Reply:
@adamrusted Great work! I've now used the new SVG that you've provided.

Reply:
Fab - did you want to update the source to the SVG also?

Reply:
Quoting @ericcornelissen:

> I'd say it can be used as a starting point, but not as an official source

Reply:
Is the precision of 4 necessary on this?

Reply:
@adamrusted It's optimized with a precision of 3, isn't it? 🤔

Reply:
Yes. Don't mind me. Reading the preview from when I left the original comment 😅